microcloud/cmd/microcloud/main_init.go

```diff
@@ -31,12 +31,14 @@ type InitSystem struct {
 
 	AvailableDisks []lxdAPI.ResourcesStorageDisk // Disks as reported by LXD.
 
-	MicroCephDisks     []cephTypes.DisksPost                  // Disks intended to be passed to MicroCeph.
-	TargetNetworks     []lxdAPI.NetworksPost                  // Target specific network configuration.
-	TargetStoragePools []lxdAPI.StoragePoolsPost              // Target specific storage pool configuration.
-	Networks           []lxdAPI.NetworksPost                  // Cluster-wide network configuration.
-	StoragePools       []lxdAPI.StoragePoolsPost              // Cluster-wide storage pool configuration.
-	StorageVolumes     map[string][]lxdAPI.StorageVolumesPost // Cluster wide storage volume configuration.
+	MicroCephDisks                 []cephTypes.DisksPost                  // Disks intended to be passed to MicroCeph.
+	MicroCephPublicNetworkSubnet   string                                 // Subnet for the Ceph public network.
+	MicroCephInternalNetworkSubnet string                                 // Subnet for the Ceph internal network.
+	TargetNetworks                 []lxdAPI.NetworksPost                  // Target specific network configuration.
+	TargetStoragePools             []lxdAPI.StoragePoolsPost              // Target specific storage pool configuration.
+	Networks                       []lxdAPI.NetworksPost                  // Cluster-wide network configuration.
+	StoragePools                   []lxdAPI.StoragePoolsPost              // Cluster-wide storage pool configuration.
+	StorageVolumes                 map[string][]lxdAPI.StorageVolumesPost // Cluster wide storage volume configuration.
```

Comment:
Can you please move all of these comments to the line above the field definition and begin them with the field name so that they are Godoc compatible?

Reply:
fixed